src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -498,31 +431,26 @@ GenericTracepointPerfEvent ConsumeGenericTracepointPerfEvent(PerfEventRingBuffer
 
 SchedWakeupPerfEvent ConsumeSchedWakeupPerfEvent(PerfEventRingBuffer* ring_buffer,
                                                  const perf_event_header& header) {
-  ORBIT_CHECK(header.size >= sizeof(perf_event_raw_sample_fixed));
-  perf_event_raw_sample_fixed ring_buffer_record;
-  ring_buffer->ReadRawAtOffset(&ring_buffer_record, 0, sizeof(perf_event_raw_sample_fixed));
+  perf_event_attr flags{};
+
+  flags.sample_type = PERF_SAMPLE_RAW | SAMPLE_TYPE_TID_TIME_STREAMID_CPU;
```

Comment:
Similar to above, Can I ask for a comment here about why these 2 are the only needed for sample_type?